search-relevancy-test/src/search_relevancy_test/relevancy_test.clj

```diff
@@ -96,8 +90,8 @@
     (println "Creating providers")
     (ingest/create-providers test-files)
     (println "Ingesting community usage metrics and test collections")
-    (ingest/ingest-community-usage-metrics) ;; Needs to happen before ingest
-    (ingest/ingest-test-files test-files)))
+    (ingest/ingest-community-usage-metrics))) ;; Needs to happen before ingest
+    ;(ingest/ingest-test-files test-files)))
```

Comment:
Is this just being temporarily commented out?

Reply:
That was a mistake.